source/_integrations/nanoleaf.markdown

```diff
@@ -12,7 +12,7 @@ ha_platforms:
 
 ### Configuration Sample
 
-The `nanoleaf` platform allows you to control [Nanoleaf Light Panels](https://nanoleaf.me) from Home Assistant.
+The `nanoleaf` platform allows you to control [Nanoleaf Light Panels](https://nanoleaf.me) from Home Assistant. Note that for full control of nanoleaf devices, particularly effects, make sure they are set up as `- platform: nanoleaf` not as a HomeKit device.
```

Comment:
```suggestion
The Nanoleaf integration allows you to control [Nanoleaf Light Panels](https://nanoleaf.me) from Home Assistant. Note that for full control of Nanoleaf devices, particularly effects, make sure they are set up as instructed on this page and not as a HomeKit device.
```

Additionally, I think the first sentence should be moved above the first heading.